src/runtime/server/plugins/refresh-token.server.ts

```diff
@@ -0,0 +1,39 @@
+import { _fetch } from '../../utils/fetch'
+import { jsonPointerGet } from '../../helpers'
+import { defineNuxtPlugin, useAuthState } from '#imports'
+export default defineNuxtPlugin({
+  name: 'refresh-token-plugin',
+  enforce: 'pre',
+  async setup (nuxtApp) {
+    // TODO: get rid of duplicate code
+    const { rawToken, rawRefreshToken, refreshToken, lastRefreshedAt } = useAuthState()
+    if (refreshToken.value) {
+      const config = nuxtApp.$config.auth
```

Comment:
I think that you forgot to put `.public.` level for the path. But actually, you can use `useRuntimeConfig().public.auth` like in the `/src/runtime/plugin.ts`